src/lib/y2partitioner/device_graphs.rb

```diff
@@ -11,37 +11,22 @@ class DeviceGraphs
     # Run a block with a **duplicate** of @dg.
     # If the block returns a truthy value,
     # the new dg is copied to the old one
-    # (which is useful if the orig one was the `staging` one)
+    # (which is useful if the orig one was the `current` one)
```

Comment:
OK, @teclator addresses these in #12. (And I agree with `create_instance` but disagree with the `Devicegraph` spelling)